### PR TITLE
Update current_activity_header.html

### PR DESCRIPTION
**Pull request by zobe123**
dynamic "s" at the word stream/streams
added "no" before Activity when their is no activity

### CODE REVIEW

**Comment by zobe123** [comment]
Maybe there is no need to add the "no" ?!

**Comment by JonnyWong16** [comment]
Yeah, no need for the "no" since it already says "Nothing is currently being watched" beneath.

File: data/interfaces/default/current_activity_header.html [hunk 1]
```diff
@@ -15,11 +15,15 @@
...
 % if data != None:
-% if data == '0':
-<h3>Activity</h3>
-% else:
-<h3>Activity <small>${data} stream(s)</small></h3>
-% endif
+	% if data == '0':
+	<h3>Activity</h3>
+	% else:
+		% if data == '1':
```
Comment (JonnyWong16) [inline data/interfaces/default/current_activity_header.html:21]: You can simplify by using else if:

```
% elif data == '1':
<h3>Activity <small>${data} stream</small></h3>
% else:
<h3>Activity <small>${data} streams</small></h3>
% endif
```

**Comment by zobe123** [comment]
i didn't know enough about the language you used at this Point. I only copy/pasted the working code... feel free to do the changes you want to have.

**Comment by JonnyWong16** [comment]
For your future reference cb7ba7f, and the language is Mako.

**Comment by zobe123** [comment]
thx

**Pull request merged**